### PR TITLE
Some restructure of configs with ttalps_samples_list and removed some histograms

### PR DESCRIPTION
**Pull request by kerstinlovisa**
I renamed config names for clarification.

--files_configs are:
ttalps_skimmer_files_config, ttalps_skimmer_das_files_config, ttalps_histogrammer_files_config

Removed some histograms and ended up adding some more flags for the histogram config.

### CODE REVIEW

File: utils/merge_histograms.py [hunk 1]
```diff
@@ -30,33 +33,34 @@ def main():
...
+    # "backgrounds2018/ttHTobb",
+    # "backgrounds2018/ttHToNonbb",
+    # "backgrounds2018/ttHToMuMu",
     
-    "backgrounds2018/TTZZ",
-    "backgrounds2018/TTZH",
-    "backgrounds2018/TTTT"
+    # "backgrounds2018/TTZZ",
+    # "backgrounds2018/TTZH",
+    # # # "backgrounds2018/TTTT"
```
Comment (jniedzie) [inline utils/merge_histograms.py:64]: Shouldn't we include these paths from the new file you created?

File: configs/ttalps_skimmer_files_config.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+# output_skim = "skimmed_ttbarSemimuonicCR_Met50GeV_1mediumBjets_muonIdIso_goldenJson"
+
+# output_skim = "skimmed_ttZSemimuonicCR_tightMuon_noLooseMuonIso"
+# output_skim = "skimmed_ttZSemimuonicCR_Met50GeV"
+
+# SRs
+# output_skim = "skimmed_looseSemimuonic_SRmuonic_DR"
+# output_skim = "skimmed_looseSemimuonic_SRmuonic_OuterDR"
+# output_skim = "skimmed_looseSemimuonic_SRmuonic_Segmentv1"
+
```
[the remark this answers is not in the document]
Comment (kerstinlovisa) [reply configs/ttalps_skimmer_files_config.py:38]: Yeah the CRs are what you put there a long time ago, which ones are still being used?

File: utils/merge_histograms.py (hunk 1 above)
Comment (kerstinlovisa) [reply utils/merge_histograms.py:64]: Oh yeah of course! didn't think about that

File: configs/ttalps_skimmer_files_config.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (jniedzie) [reply configs/ttalps_skimmer_files_config.py:38]: Hmm, I guess we can keep just a few for reference, but we will need to rerun them on LLPnanoAOD anyway. Maybe keep:

```
"skimmed_ttbarSemimuonicCR_Met50GeV_1mediumBjets_muonIdIso_goldenJson"
"skimmed_ttZSemimuonicCR_Met50GeV"
```

**Pull request merged**